### DISCUSSION TITLE
Is possible to run x86 binaries in termux? (arm)

### DISCUSSION THREAD

**Discussion by Luizin5** [termux/termux-app#2578] [Q&A] [answered]
Hello, i'm want to know if i can execute x86 binaries in termux, and how I do it.
I tried with qemu-i386, but it's returning " permission denied" (i ran the command with superuser permissions)

**Comment by sylirre** [comment, marked as answer]
The binary must be `chmod +x ./yourfile` before it can be executed.

Do not run binaries on shared storage (/sdcard) because it is mounted as with noexec option.
